src/test/java/utilities/Util.java

```diff
@@ -16,3 +18,19 @@ public static void wait(int seconds) {
 	}
 
 }
+	/*
+	 * Hopefully rhis should work
+	 * Switches the driver focus between the windows. 
+	 * Author Jolene Jones 
+	 * 2nd edition 
+	 * Published 2/8/2023
+	 */
+
+	public static void switchTo(String pageURL, WebDriver driver) {
```

Comment:
1, you switch window by window handle, not page url, so this function doesn't work
2, too much unnecessary comments
3, Formatting is off

Reply:
The changes have been made